packages/erlang/prefix.patch

```diff
@@ -1,14 +0,0 @@
---- ./lib/kernel/src/os.erl	2022-05-17 19:29:45.000000000 +0000
-+++ ./lib/kernel/src/os.erl	2022-05-18 14:34:43.819497497 +0000
-@@ -317,9 +317,9 @@
-     %% Check if the default shell is located in /bin/sh as expected usually
-     %% or in /system/bin/sh as implemented on Android. The raw option is
-     %% used to bypass the file server and speed up the file access.
--    Shell = case file:read_file_info("/bin/sh",[raw]) of
-+    Shell = case file:read_file_info("@TERMUX_PREFIX@/bin/sh",[raw]) of
-                 {ok,#file_info{type=regular}} ->
--                    "/bin/sh";
-+                    "@TERMUX_PREFIX@/bin/sh";
```

Comment:
How come this patch isn't needed anymore, did upstream change the way this is handled?

Reply:
They patched it on the upstream, with `/system/bin/env` assuming `/bin/env` doesn't exist.

Reply:
Upstream patch: https://github.com/erlang/otp/blob/3ab59bbd6cfea08c1450feed11bd50e3487fe21b/lib/kernel/src/os.erl#L665-L683